testdata/runtest.sh

```
@@ -24,9 +24,9 @@ go run ./cmd/prestd/main.go migrate up --path ./testdata/migrations
 
 echo "\n\n.:: PRESTD: TESTING STARTING..."
 if [ -z ${1+x} ]; then
-    go test -v -race -failfast -covermode=atomic -coverprofile=coverage.out ./...;
+    go test -v -race -failfast ./integration/...;
 else
-    go test -v -race -failfast -covermode=atomic -coverprofile=coverage.out $@
+    go test -v -race -failfast $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)